src/VisualStudio/Core/Def/Implementation/LanguageService/AbstractPackage`2.cs

```diff
@@ -90,19 +81,19 @@ protected override async Task InitializeAsync(CancellationToken cancellationToke
 
         protected override async Task LoadComponentsAsync(CancellationToken cancellationToken)
         {
+            var workspace = ComponentModel.GetService<VisualStudioWorkspace>();
+
             await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
 
             // Ensure the nuget package services are initialized after we've loaded
             // the solution.
-            _packageInstallerService = Workspace.Services.GetService<IPackageInstallerService>() as PackageInstallerService;
-            _symbolSearchService = Workspace.Services.GetService<ISymbolSearchService>() as VisualStudioSymbolSearchService;
+            _packageInstallerService = workspace.Services.GetService<IPackageInstallerService>() as PackageInstallerService;
```

Comment:
Do these need to be initialized (and connected) in the non-package load paths as well?

Reply:
For now, probably not. These are used for figuring out code fixes, but we won't really suggest fixes until a file is opened, and that would trigger the package load because it'll load the editor factory. If that all gets fixed then yes, this is problematic.

I'll add a comment to better indicate when this shouldn't be there.